container/shared_network.go

```
@@ -27,25 +24,23 @@ func createSharedNetworkIfNeeded(client *docker.Client) (err error) {
 			"com.docker.stack.namespace": namespace,
 		},
 	})
-	return
+	return nil
 }
 
-// sharedNetwork returns the shared network created to connect services and MESG Core
-func sharedNetwork(client *docker.Client) (network types.NetworkResource, err error) {
-	network, err = client.NetworkInspect(context.Background(), Namespace(sharedNetworkNamespace), types.NetworkInspectOptions{})
-	return
+// sharedNetwork returns the shared network created to connect services and MESG Core.
+func sharedNetwork(client *docker.Client) (types.NetworkResource, error) {
+	return client.NetworkInspect(context.Background(), Namespace(sharedNetworkNamespace), types.NetworkInspectOptions{})
 }
 
-// SharedNetworkID returns the id of the shared network
-func SharedNetworkID() (networkID string, err error) {
+// SharedNetworkID returns the ID of the shared network.
+func SharedNetworkID() (string, error) {
 	client, err := Client()
 	if err != nil {
-		return
+		return "", nil
```

we need to return the error

---

Fixed